models/intel/dns-poconet-like-0001/dns-poconet-like-0001.md

```diff
@@ -0,0 +1,167 @@
+# dns-poconet-like-0001
+
+## Use Case and High-Level Description
+
+This is a PoCoNet like model for noise suppression.
+The model is based on PoCoNet architecure described in [1] and trained on clean and noise samples from DNS-chalange dataset [2,3].
+The network works with mono audio sampled on 16kHz.
+The audio processed iterative by patches with 2048 size.
+On each iteration it takes 2048 (128ms) samples as input and returns 2048 (128ms) samples as output with 640 (40ms) samples delay.
+In addition the network required 52 state tensors to make processing seamless.
+On the first iteration these state tensors have to be filled by 0.
+On the consequences iterations theses tensors have to be taken from corresponding outputs of previous iteration.
+You can try demo (../../../demos/deep_noise_suppresion/python) to see how it works.
+
+## Specification
+
+to process 2048 samples that is 128ms for 16kHz
+
+| Metric            | Value                 |
+|-------------------|-----------------------|
+| GOps              | 1.2                   |
+| MParams           | 7.22                  |
+| Source framework  | PyTorch\*             |
+
+
+
+## Accuracy
+
+The SISDR [5] quality metric was calculated on the 100 dev test synthetic speech clips from DNS-chalange dataset [4].
+
+
+| Metric                    | Value         |
+|---------------------------|---------------|
+| SISDR OUT                 |    20.23   dB |
+| SISDR DIFF                |     8.49   dB |
+
+
+## Input
+
+Sequence patch, name: `input`, shape: [1, 2048], format: [B, T]
```

Comment:
```suggestion
Sequence patch, name: `input`, shape: `1, 2048`, format: `B, T`
```

Reply:
please align other inputs/outputs shape and layout description to use suggested format